docs/editor/workspaces.md

```diff
@@ -9,39 +9,91 @@ MetaDescription: Learn about Visual Studio Code workspaces
+You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. Think of a workspace as the root of a project that comes with extra VS Code knowledge and capabilities.
 
-## Workspace state
+## How do I open a VS Code "workspace"
+
+You don't have to do anything for a folder to become a VS Code workspace other than open the folder with VS Code. Once a folder or has been opened, VS Code will automatically keep track of things such as your open files and editor layout so the editor will be as you left it when you reopen that folder. You can also manually add other folder specific configurations such as workspace-specific settings (versus global user settings) and task definition and debugging launch files.
+
+![Single-folder workspace](images/workspaces/single-folder-workspace.png)
+
+*A single folder workspace opened inside VS Code*
+
+The same is true for multi-root workspaces that you can open easily by selecting the `<name>.code-workspace` file for opening.
+
+![Multi-folder workspace](images/workspaces/multi-folder-workspace.png)
 
+*A multi-root workspace opened in VS Code*
+
+**Note:** The visual difference of having a folder opened vs. opening a `.code-workspace` file can be very subtle. To give you a hint that a `.code-workspace` file has been opened, some pieces of the user interface (e.g. the root of the explorer) show an extra "(Workspace)" suffix next to the name.
+
+## Workspace state
+**bpasero>** I would leave this paragraph out as we already gave some examples of workspace state before
 List some of the metadata and state persisted per folder TBD
 
 ## Workspace settings
-
+**bpasero>** see my note in [1]
 Workspace files are stored at the project root in a `.vscode` folder. TBD
 
 ### settings
-
+**bpasero>** see my note in [1]
 Describe workspace settings, Settings UI tab, why you would use workspace settings versus user settings
 
 ### debugging
-
+**bpasero>** see my note in [1]
 Briefly describe `launch.json` and link to other docs TBD
 
 ### tasks
-
+**bpasero>** see my note in [1]
 Briefly describe `tasks.json` and link to other docs TBD
 
+[1] **bpasero>**
+I am trying to distill what we need to explain for settings, debugging and tasks in the context of explaining what a "workspace" is. My initial reaction is that all of this should have already been explained in the respective doc pages so I would not make this very detailed here to not repeat everything. However, I think one important concept is worthwhile to mention here: if you open a `.code-workspace` file (or actually when you are inside an "untitled" workspace), you get a third dimension of settings that you don't have otherwise:
```

Comment:
I think there is some value in mentioning the .vscode folder, which can include files such as settings.json, tasks.json, launch.json, extensions.json. Agreed that we don't need to go into too much detail.
Perhaps we can just have a bulleted list mentioning the files with a brief description of what they control with a link to the appropriate main doc.
Also have a quick mention that in a multi-root workspace there can be multiple .vscode folders contributing configurations.